herddb-core/src/main/java/herddb/sql/CalcitePlanner.java

```diff
@@ -883,8 +886,8 @@ private PlannerOp planEnumerableHashJoin(EnumerableHashJoin op, RelDataType rowT
             fieldNames[i] = col.name;
             columns[i++] = col;
         }
-        return new SemiJoinOp(fieldNames, columns,
```

Comment:
If we are not using this class anymore you can drop it.

Reply:
Ok.